tests/unit/segments/fixtures.py

```diff
@@ -148,3 +149,23 @@
         )
     ],
 )
+segment_identity_override = SegmentModel(
+    id=7,
+    name="segment_identity_override",
```

Comment:
We talked about this being unique in someway, right? Should we illustrate that even in this test data?

Reply:
Done.